react/features/base/lib-jitsi-meet/functions.native.ts

```diff
@@ -1,9 +1,8 @@
-// @flow
-
+// @ts-ignore
```

Comment:
why in the whole file?

Reply:
?
Bourne is used in the `loadConfig` function below. And ts-ignore is needed because this lib doesn't have types

Reply:
👍